### ISSUE TITLE
"Client is not recording error" on mobile Chrome on Android

### ISSUE THREAD

**Issue by samutamm** [speechly/browser-client#31]
### Describe the bug
I tried [browser-client-example](https://speechly.github.io/browser-client-example/) on desktop and it works fine. But on my mobile when I'm speaking while holding down the "Record" button, nothing happens on the screen.

### To Reproduce
1. Go to https://speechly.github.io/browser-client-example/
2. Click on "Connect" button
3. Click on "Record" button and hold it down
4. Say "turn the lights on in the kitchen"
5. Release the "Record" button
6. There is no output on the screen

### Expected behaviour
When I repeat the same steps on desktop Chrome, I can see the transcriptions, intents and entities at steps 4 & 5.

### Screenshots
Here's a couple of screenshot from _inspect view_ (using `chrome://inspect/#devices`), connected to my mobile.

After loading the page
<img width="604" alt="Screenshot 2020-06-18 at 10 39 50" src="https://user-images.githubusercontent.com/7460037/84992169-3dc45b00-b150-11ea-8b1b-5c679e4ddfca.png"> 

Then after clicking the "Connect" button

<img width="604" alt="Screenshot 2020-06-18 at 11 32 21" src="https://user-images.githubusercontent.com/7460037/84997924-d4e0e100-b157-11ea-9938-545d1043b898.png">

And finally after pushing down Record button, talked and released Record button:

<img width="604" alt="Screenshot 2020-06-18 at 11 35 13" src="https://user-images.githubusercontent.com/7460037/84998001-ec1fce80-b157-11ea-8999-05b6b4489477.png">

On the left, at the bottom of the mobile screen there should be visible the recognized transcripts etc, but it is empty.

The log output in the console is
```
browser_audio_processor.ts:51 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu
client.ts:462 [SpeechlyClient] State transition Disconnected Connecting
client.ts:462 [SpeechlyClient] State transition Connecting Connected
client.ts:462 [SpeechlyClient] State transition Connected Starting
client.ts:462 [SpeechlyClient] State transition Starting Recording
client.ts:405 [SpeechlyClient] Server connection closed Error: Websocket was closed: The server has encountered an unexpected error
    at WebSocket.onWebsocketClose (websocket_client.ts:141)
handleWebsocketClosure @ 2.5afbdc05.chunk.js:1
onWebsocketClose @ 2.5afbdc05.chunk.js:1
client.ts:462 [SpeechlyClient] State transition Recording Connecting
client.ts:427 [SpeechlyClient] Attempting to re-connect to the server in 1000ms
client.ts:427 [SpeechlyClient] Attempting to re-connect to the server in 2000ms
index.ts:200 Could not stop recording Error: Cannot stop context - client is not recording
    at e.value (client.ts:244)
    at HTMLButtonElement.a (index.ts:198)
client.ts:427 [SpeechlyClient] Attempting to re-connect to the server in 4000ms
client.ts:427 [SpeechlyClient] Attempting to re-connect to the server in 8000ms
client.ts:427 [SpeechlyClient] Attempting to re-connect to the server in 16000ms
client.ts:462 [SpeechlyClient] State transition Connecting Disconnected
```

### Environment

- Platform: Mobile (Huawei P20 EML-L29)
- OS: Android 9
- Browser: Chrome
- Version: 83.0.4103.106
- Package version: browser-client@0.5.1

### Additional context
-

**Comment by aukkonen** [comment]
Now running iPhone SE, iOS 13.5.1, not seeing this bug on Safari.

**Comment by nori266** [comment]
Seems that I have the same error as Samu has
![image](https://user-images.githubusercontent.com/20275582/85002319-6bfc6780-b15d-11ea-83ee-b70360931754.png)

Platform: Mobile (Samsung Galaxy A40)
OS: Android 10
Browser: Chrome
Version: 80.0.3987.99

**Comment by radiohead** [comment]
Fixed in 1.0.0
